mutator/statement/remove.go

```diff
@@ -100,6 +98,48 @@ func (m *MutatorRemoveStatement) Mutate(node ast.Node, changed chan bool) {
 	}
 }
 
+func createNoop(old ast.Stmt) ast.Stmt {
+	v := &idCollector{}
+	ast.Walk(v, old)
+	return v.generateStatement(old)
+}
+
+type idCollector struct {
+	Ids []ast.Expr
+}
+
+func (i *idCollector) Visit(node ast.Node) ast.Visitor {
+	switch v := node.(type) {
+	case *ast.Ident:
```

Comment:
"main_test.go" in "github.com/zimmski/go-mutesting/cmd/go-mutesting" reveals that it is not enough to filder for identifiers, you also need to filter out calls, or else they are substituted too. E.g.

```diff
--- example.go  2015-01-03 14:11:26.000000000 +0100
                        +++ /tmp/go-mutesting-363277582/example.go.11   2016-06-22 15:47:24.236065835 +0200
                        @@ -24,7 +24,7 @@
                                n += bar()
                         
                                bar()
                        -       bar()
                        +       _ = bar
                         
                                switch {
                                case n < 20:
```